tests/TestCase/Command/PluginListCommandTest.php

```diff
@@ -0,0 +1,94 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP :  Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP Project
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Test\TestCase\Command;
+
+use Cake\Console\CommandInterface;
+use Cake\Console\TestSuite\ConsoleIntegrationTestTrait;
+use Cake\TestSuite\TestCase;
+
+/**
+ * PluginListCommandTest class.
+ */
+class PluginListCommandTest extends TestCase
+{
+    use ConsoleIntegrationTestTrait;
+
+    /**
+     * setUp method
+     */
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->setAppNamespace();
+    }
+
+    /**
+     * Test generating help succeeds
+     */
+    public function testHelp(): void
+    {
+        $this->exec('plugin list --help');
+        $this->assertExitCode(CommandInterface::CODE_SUCCESS);
+        $this->assertOutputContains('plugin list');
+    }
+
+    /**
+     * Test plugin names are being displayed correctly
+     */
+    public function testList(): void
+    {
+        $configPath = ROOT . DS . 'cakephp-plugins.php';
+        $this->skipIf(file_exists($configPath), 'cakephp-plugins.php exists, skipping overwrite');
```

Comment:
I wonder if this instead of skipping the test should maybe unlink() the file instead?
Or globablly in setUp hook?

Reply:
I copied this from another test using the cakephp-plugins.php but sure, I'll try to clean that up